### DISCUSSION TITLE
Index link button not showing on telegram chat

### DISCUSSION THREAD

**Discussion by keerthyvash** [anasty17/mirror-leech-telegram-bot#1323] [Q&A] [answered]
![Uploading IMG_20230103_201549.jpg…]()
In my config.env file the VIEW_LINK = "True"
And even the INDEX _URL = "https://example.workers.dev/0:"

The files is getting properly download and saved on Google drive also getting drive link in the telegram bot . But the index link button 
![IMG_20230103_201549](https://user-images.githubusercontent.com/115761173/210380277-2cd26ef3-38cd-4750-b1eb-f49fe693ae4c.jpg)
is not showing in the telegram bot 

What could be the possible fix ?

**Comment by anasty17** [comment, marked as answer]
If DATABASE_URL added then edit the var from bot settings and not from config.env
